scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -56,13 +56,14 @@ def get_machine_arch():
     return os.uname()[4]
 
 
-def get_os_loader():
-    path = '/usr/share/qemu/aavmf-aarch64-code.bin'
+def get_os_loader(firmware_type=None):
+    path = None
+    template = "<loader readonly='yes' type='pflash'>%s</loader>"
     if 'aarch64' in get_machine_arch():
-        return """
-  <loader readonly='yes' type='pflash'>%s</loader>
```

Comment:
for both aarch64 and uefi with x86_64, we still return a valid element [here](https://github.com/SUSE-Cloud/automation/pull/1141/files/6402e6477e9abc5502b683475e4579d8192f040b#diff-c4e86c61b1673e20ce3773716936b40cR66) . Just the logic has changed due to an earlier comment.